src/components/BugzillaComponents/index.jsx

```diff
@@ -130,6 +131,13 @@ const getTableHeaders = (data, onComponentDetails) => {
   return Headers;
 };
 
+/**
+ * @description Reversing the order according to first parameter.
+ * @param {String} order asc or dsc
+ * @returns {Number} 1 || -1
+ */
+const ascDescSortFunc = order => (order === 'desc' ? 1 : -1);
```

Comment:
can we avoid the duplication 
with 
https://github.com/mozilla/bugzilla-dashboard/pull/124/files#diff-8af48e0bb1060bc17bb21e768fb8e21dR52 ?

Reply:
Moved ascDescSortFunc function to sort.js file to avoid code duplication